ParisBNF/abb/BNFabb41.xml

```diff
@@ -0,0 +1,387 @@
+                 </msItem>
+                     </msItem>
+               </msContents>
+
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <watermark/>
+                           <material key="parchment"/>
+
+                        </support>
+                        <extent>
+                           <measure unit="page">471</measure>
+                           <measure unit="page" type="blank">1</measure>
+                           <locus target="#105r"/>
+
+                           <dimensions type="outer" unit="mm">
+                              <height>440</height>
+                              <width>385</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>
+                        Pagination by
+                        <persName ref="PRS1071dAbbadi" role="owner">Antoine d'Abbadie</persName>
+                      in the upper right corner of the pages, after <locus target="#40"/>,
+                    only on the recto-sides.</foliation>
+                        <collation>
+                        </collation>
+                        <condition key="good">Leather is
+                        slightly abraded on the corners of the boards.</condition>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="24">
+                           <locus from="1" to="471"/>
+                        </layout>
+
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">18th-century script</seg>
+                        <date notBefore="1700" notAfter="1799"/>
+                        <seg type="ink">Black, red</seg>
+                        <desc>Finely executed, large-size
+                          script (letter size 7-11 mm).</desc>
+                        </handNote>
+                  </handDesc>
+                  <decoDesc>
+                    <decoNote type="ornamentation" xml:id="d1">
+                      <locus target="471"/>
+                      <desc>Text divider at the end consisting of
+                        two red and black dotted lines and
+                        ornament of red and black dot punctuation signs.</desc>
+                    </decoNote>
+                  </decoDesc>
+
+                  <additions>
+                     <list>
+                       <item xml:id="e1">
+                        <desc type="StampExlibris">
+                          <locus target="#1"/>
+                          Stamp of the library.
+                        </desc>
+                      </item>
+
+                        <item xml:id="e2">
```

Comment:
I would classify this as an addition, not an extra.

Reply:
ok